src/core/assert/globals.d.ts

```diff
@@ -0,0 +1,8 @@
+export * from './amp-globals.d';
```

Comment:
We aren't exporting anything right? Same in a few other files.

```suggestion
import './amp-globals.d';
```

Reply:
Done